test/OpenTelemetry.Exporter.InfluxDB.Tests/Utils/LineProtocolParser.cs

```diff
@@ -94,6 +97,6 @@ private static DateTime ParseTimestamp(string? timestampSection)
         }
 
         long ticks = unixTimeNanoseconds / 100;
-        return DateTime.UnixEpoch.AddTicks(ticks);
```

Comment:
@CodeBlanch Why this change?

Reply:
`DateTime.UnixEpoch` is only available on [netstandard2.1 & netcoreapp2.1+](https://learn.microsoft.com/dotnet/api/system.datetime.unixepoch?view=net-7.0#applies-to) so I had to come up with some other solution for the .NET Framework targets on the project so it compiles. I decided just to use a local version of the value.